install.sh

```diff
@@ -50,7 +50,8 @@ if [[ "$OS" == "Darwin" ]]; then
       
       if [[ "$VERSION" != "latest" ]]; then
         ohai "Installing Agentuity CLI version $VERSION using Homebrew..."
-        brew install agentuity/tap/agentuity@$VERSION
+        FORMULA="agentuity/tap/agentuity@${VERSION}"
+        brew install $FORMULA
```

Comment:
i'm still getting an error:

No available formula or cask with the name "agentuity/tap/agentuity@"
 
Is $VERSION not set?